src/components/Select/Select.stories.mdx

```diff
@@ -88,6 +100,7 @@ is left blank, a value will be generated for you.
       isDisabled: false,
       isInvalid: false,
       isRequired: false,
+      labelPosition: "LabelPositions.Top",
```

Comment:
The `labelPosition` control isn't working in Storybook. On line 115, for the `Select` component, besides adding `{...args}` it also needs to explicitly have `labelPosition={labelPositionsEnumValues.getValue(args.labelPosition)}`. That should make it work.